website/blog/2025-11-24-aks-automatic-managed-system-node-pools/index.md

```diff
@@ -0,0 +1,172 @@
+
+AKS Automatic pricing includes a fixed monthly cluster fee and per-vCPU charges on top of standard VM compute costs. This pricing includes financially backed SLAs for both API server uptime and pod readiness. For current rates and a full breakdown by VM category, see the [Azure Kubernetes Service pricing page](https://azure.microsoft.com/pricing/details/kubernetes-service#pricing).
+
+## Getting started
+
+### Prerequisites
+
+- Azure CLI 2.77.0 or later.
+- `aks-preview` extension 19.0.0b15 or later.
+
+```bash
+# Install or update the aks-preview extension
+az extension add --name aks-preview
+az extension update --name aks-preview
+```
+
+### Register the preview feature
+
+```bash
+az feature register --name AKS-AutomaticHostedSystemProfilePreview --namespace Microsoft.ContainerService
+```
+
+### Create the cluster
+
+Select a region where managed system node pools are available. See the [regional availability](https://aka.ms/aks/automatic/managed-systempool-regions) section in the documentation for the current list.
+
+#### Set your variables
+
+```bash
+RESOURCE_GROUP="myResourceGroup"
+CLUSTER_NAME="myAKSCluster"
+LOCATION="westcentralus"  # Choose a supported region (see: https://aka.ms/aks/automatic/managed-systempool-regions)
+```
+
+#### Create the resource group
+
+```bash
+az group create --name $RESOURCE_GROUP --location $LOCATION
+```
+
+#### Create an Automatic cluster with a managed system node pool
+
+```bash
+az aks create \
+--resource-group $RESOURCE_GROUP \
+--name $CLUSTER_NAME \
+--location $LOCATION \
+--sku automatic \
+--enable-hosted-system
+```
+
+The output includes `"hostedSystemProfile": { "enabled": true }` confirming the feature is active.
+
+### Connect to the cluster and deploy an application
+
+Get credentials for your cluster and deploy the [AKS Store demo application](https://github.com/Azure-Samples/aks-store-demo):
+
+```bash
+az aks get-credentials --resource-group $RESOURCE_GROUP --name $CLUSTER_NAME
+
+kubectl create ns aks-store-demo
+kubectl apply -n aks-store-demo -f https://aka.ms/aks/quickstarts/store.yaml
+```
+
+Check the ingress address and open it in your browser once an IP is assigned:
+
+```bash
+kubectl get ingress store-front -n aks-store-demo --watch
+```
+
+![Screenshot of the deployed application on an AKS Automatic cluster](contoso-pet-store.png)
+
+Your workload runs on user node pools in your subscription that Node Auto Provisioning will create, while system services stay on the managed pool.
+
+![Screenshot of AKS desktop application showing the nodes in the cluster](aks-desktop-nodes.png)
+
+> **Tip:** Prefer a graphical experience? [AKS Desktop](https://learn.microsoft.com/azure/aks/aks-desktop-overview) lets you manage clusters, view workloads, and troubleshoot issues without leaving your desktop.
+
+The managed system nodes will not run on your Azure subscription.
+
+![Screenshot of the Azure portal showing that the managed syste nodes are not there](portal-vms.png)
+

```

Comment:
General Roadmap of things to come in AKS automatic space ?

Reply:
Link to the AKS roadmap already included below.